wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -1,222 +1,257 @@
-Song Setup
-============
+# Song Setup
 
-The Song Setup is an important screen in [beatmapping](/wiki/Beatmapping), as it contains many important settings that are keys to make a decent map.
+Song Setup contains metadata and general beatmap configuration settings.
```

Comment:
Song Setup is the fourth section of the <Beatmap Editor link here>. It contains metadata and general beatmap configuration settings.